internal/testplanet/storagenode.go

```diff
@@ -109,6 +109,8 @@ func (planet *Planet) newStorageNodes(count int, whitelistedSatelliteIDs []strin
 				StaticDir: filepath.Join(developmentRoot, "web/operator/"),
 			},
 			Storage2: piecestore.Config{
+				ExpirationGracePeriod: 0,
+				MaxConcurrentRequests: 100,
```

Comment:
Leaving testplanet to 100, because having this lower will probably break some tests.

Reply:
Worth testing where the limit is, in another PR of course.